clientgui/BOINCBaseView.cpp

```diff
@@ -250,10 +250,15 @@ void CBOINCBaseView::OnListRender(wxTimerEvent& event) {
                 m_pListPane->DeleteAllItems();
             } else {
                 int iIndex = 0;
+#if wxDEBUG_LEVEL
                 int iReturnValue = -1;
+#endif
                 if (iDocCount > iCacheCount) {
                     for (iIndex = 0; iIndex < (iDocCount - iCacheCount); iIndex++) {
-                        iReturnValue = AddCacheElement();
+#if wxDEBUG_LEVEL
+                        iReturnValue =
```

Comment:
Here's the same as before: I strongly suggest to check return value in Release mode also, and do this properly, otherwise in Release we might have unexpected results.

Reply:
This is not my code. Rom Walton wrote it many years ago. While I agree with you, it is well beyond the scope of this PR and not something I am willing to do. If you feel strongly, I suggest you create a separate issue.